### PR TITLE
Add start/finish time columns to job list

### PR DESCRIPTION
**Pull request by vimagick**
Displaying start/finish time on job list is very handy for schedule checking.

screen shot 2017-01-14 at 01 06 47

### CODE REVIEW

File: scrapyd/website.py [hunk 1]
```diff
@@ -140,7 +142,9 @@ def render(self, txrequest):
             for a in ['project', 'spider', 'job']:
                 s += "<td>%s</td>" % getattr(p, a)
             s += "<td></td>"
-            s += "<td>%s</td>" % (p.end_time - p.start_time)
+            s += "<td>%s</td>" % p.start_time.replace(microsecond=0)
+            s += "<td>%s</td>" % p.end_time.replace(microsecond=0)
+            s += "<td>%s</td>" % (p.end_time.replace(microsecond=0) - p.start_time.replace(microsecond=0))
```
Comment (Digenis) [inline scrapyd/website.py]: `(p.end_time - p.start_time).replace(microsecond=0)`
floor/round/ceil for display purposes should only happen at the end of the calculation

File: scrapyd/website.py [hunk 2]
```diff
@@ -129,7 +129,9 @@ def render(self, txrequest):
             s += "<tr>"
             for a in ['project', 'spider', 'job', 'pid']:
                 s += "<td>%s</td>" % getattr(p, a)
-            s += "<td>%s</td>" % (datetime.now() - p.start_time)
+            s += "<td>%s</td>" % p.start_time.replace(microsecond=0)
+            s += "<td></td>"
+            s += "<td>%s</td>" % (datetime.now().replace(microsecond=0) - p.start_time.replace(microsecond=0))
```
Comment (Digenis) [inline scrapyd/website.py]: https://github.com/scrapy/scrapyd/pull/202/commits/5893965106b1ee6947ed42be01e647fbcf1d91dc#r96112811 applies here too

**Comment by Digenis** [comment]
Hi @vimagick,
thank you for your contribution.

I think "start, runtime, finish" is a more intuitive column order.
I'm not sure I mind the missing microseconds
as long as they are available in the json api (the webservices).
I do use a similar patch and find it very useful.

We should remember to consider adding the enqueue time,
if we start tracking it too. Currently we don't.

What do other project member think?
Should we include it in 1.2?

File: scrapyd/website.py (hunk 1 above)
Comment (vimagick) [inline scrapyd/website.py]: @Digenis No such method: `datetime.timedelta.replace`

Comment (Digenis) [inline scrapyd/website.py]: Well, python just surprised me in a bad way :confused:
We could solve this with a rounding/truncating function
but let's not bother. The current precision suffices.

**Comment by Digenis** [comment]
Will merge if no comments.
@vimagick, how about putting the runtime column before the finish column?

**Comment by vimagick** [comment]
@Digenis No problem 👌

**Pull request merged**